docs/research/repo/background-compounding-triggers-repo-audit.md

```
@@ -0,0 +1,621 @@
+# Repo Audit: Background Compounding Triggers
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)